webanno-ui-project/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/project/ProjectUsersPanel.java

```diff
@@ -377,6 +404,7 @@ public UserDetailForm(String id)
                 @Override
                 public void onSubmit()
                 {
+                	filterText.setModel(Model.of(""));
```

Comment:
I've changed it so that now the userFilter field is being changed:
userSelectionModel.userFilter = "";
But now the field does not clear its previous value after clicking "Add User"